content/gocourse-2019-05-melbourne.slide

```diff
@@ -908,20 +903,18 @@ Embedding types introduces the possiblity for naming conflicts. Go follows some
 		return s
 	}
 - Error values can be logged and printed:
-	if _, err := Sqrt(-1); err != nil {
-		log.Println(err)
-	}
+    log.Println(err)
 
 * Error Handling
 
-- Check for the presence of an error by [[https://goplay.space/#40Ho_HWiYrb][comparing with nil]]
-- [[https://goplay.space/#T2RkoxxpuUV][Comparing error values]]
-- Errors are values so they can be [[https://goplay.space/#F_EG5jM_fZ6][programmed]]
-- [[https://goplay.space/#bTOIkCkGssE][Handle errors gracefully]]
-- Check errors using [[https://goplay.space/#ISKTAUFOyjk][type assertions]]
+- Check for the [[https://goplay.space/#X0qqmcrO6om][presence of an error]]
+- Comparing [[https://goplay.space/#HiLw8Y_D_6T][error values]]
+- Checking error types using [[https://goplay.space/#CII8be9-TqW][type assertions]]
+- Errors are values so they can be [[https://goplay.space/#I6UJNHe6C7v][programmed]]
```

Comment:
IMO less words are easier to parse on slides and more consistent with existing one.
I'd use as full links with nothing else around it:
- Check for error
- Compare error values
- Compare error types
- Program with error